### DISCUSSION TITLE
how to set multi domain using .env file

### DISCUSSION THREAD

**Discussion by yongsin0129** [odoocker/odoocker#3] [Q&A] [answered]
thank you for creating this so convenient compose file. 
i really appreciate your project , it help me to build a docker service very quickly.

and now , I have two domain need to be set into nginx.

I try DOMAIN=foo.example.com boo.example.com

but it doesn`t work , how can I set multi domain by .env 

or is this another way to achieve my goal ? 

thank you

**Comment by yhaelopez** [comment, marked as answer]
I really appreciate your words and the time you took to test **Odoocker**!
The magic of multi-domain relays in the **nginx-proxy** container configuration.

I'll work around this feature and follow up here for the updates on this feature.

**Comment by yongsin0129** [comment]
thank you for your help.

I`m looking forward to see this feature in the future.
